ee/cis/macos-14/test/scripts/not_always_working_CIS_2.10.1.sh

```
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 #replace username
-sudo -u <username> /usr/bin/defaults -currentHost write com.apple.screensaver idleTime -int 1200
\ No newline at end of file
+sudo -u <username> /usr/bin/defaults -currentHost write com.apple.screensaver idleTime -int 900
```

Shellcheck (0.11.0)

[info] sudo doesn't affect redirects. Use sudo cat file | ..

(SC2024)